hack/build-go.sh

```diff
@@ -30,5 +28,19 @@ fi
 
 mkdir -p ${BIN_PATH}
 
-echo "Building ${REPO} (${VERSION_OVERRIDE})"
+# Build the cluster-version-operator-tests binary and compress it
+echo "Building ${REPO} cluster-version-operator-tests binary (${VERSION_OVERRIDE})"
+GO_COMPLIANCE_POLICY="exempt_all" CGO_ENABLED=0 GOOS=${GOOS} GOARCH=${GOARCH} \
+go build                                                                      \
+${GOFLAGS}                                                                    \
+-ldflags "${GLDFLAGS}"                                                        \
+-o "${BIN_PATH}/cluster-version-operator-tests"                               \
+"${REPO}/cmd/cluster-version-operator-tests/..."
```

Comment:
nit: make some indent for readability: they are the lines from a long cmd starting from the above.

```suggestion
  go build                                                                      \
  ${GOFLAGS}                                                                    \
  -ldflags "${GLDFLAGS}"                                                        \
  -o "${BIN_PATH}/cluster-version-operator-tests"                               \
  "${REPO}/cmd/cluster-version-operator-tests/..."
```

Reply:
Thanks, looks much better. Done.